nilrt_snac/opkg.py

```diff
@@ -55,24 +55,38 @@ def install(  # noqa: D102 - Missing docstring in public method (auto-generated
         else:
             logger.debug(f"{package} already installed")
 
-    def remove(  # noqa: D102 - Missing docstring in public method (auto-generated noqa)
-        self, package: str, ignore_installed=False, force_essential: bool = False, force_depends: bool = False
+    def remove(
+        self,
+        package: str,
+        autoremove: bool = False,
+        ignore_installed = False,
+        force_essential: bool = False,
+        force_depends: bool = False,
     ) -> None:
+        """Remove a single package from the system."""
```

Comment:
Sorry if this is pedantic, but it seems to me like this could remove more than a single package.

Reply:
Agreed; will fix.